README.md

```diff
@@ -0,0 +1,175 @@
+
+# MUSIC-LIBRARY
+
+Backend API and database for storing artist & album data. Both tables linked by artistId as a foreign key in Albums. Completed as part of the Command Shift Full Stack Bootcamp, March'23 cohort.
+
+
+## Installation
+
+```bash
+git clone https://github.com/HollyJM81/music-library.git
+```
+```bash
+npm install
+```
+
+Create .env and .env.test files
+Enter your database's credentials in both (make sure PGDATABASE name in .env.test is different)
```

Comment:
These are appearing on the same line. If you want to see what this would lookl ike before committing it and pushing it up to github, you can right click on the readme tab in the IDE and select 'open preview', you can play about with it a bit more locally that way
```
Create .env and .env.test files

Enter your database's credentials in both (make sure PGDATABASE name in .env.test is different)
```